### PR TITLE
feat: [HL-3] Add splash screen 🍕

### PR DESCRIPTION
**Pull request by ahmdmhasn**
Reopen https://github.com/motoon-eg/healthy/pull/24

Description:
Creating Splash Screen🍕.

Changes were made:
Add Splash Screen classes.
Add Design to xib file 🎨.

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ force_unwrapping: error
 force_try:
   severity: error
 
-line_length: 100
+line_length: 150
```
Comment (ahmdmhasn) [inline .swiftlint.yml:52]: Please revert this change. We can update it to another pull request to 120

File: Healthy/Classes/Generated/UIImage.Generated.swift [hunk 2]
```diff
@@ -9,4 +9,12 @@ extension UIImage {
     static var accentColor: UIImage {
       UIImage(named: "AccentColor")!
     }
+
+    static var backgroundSplash: UIImage {
+      UIImage(named: "background-splash")!
+    }
+
+    static var iconSplash: UIImage {
+      UIImage(named: "icon-splash")!
```
Comment (ahmdmhasn) [inline Healthy/Classes/Generated/UIImage.Generated.swift:18]:
```suggestion
      UIImage(named: "background-splash")!
    }

    static var iconSplash: UIImage {
      UIImage(named: "icon-splash")!
```

This is the naming convention we need to follow 
icon-...
background-...

File: Healthy/Classes/Modules/Splash/SplashViewController.swift [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    }
+}
+
+// MARK: - Actions
+
+private extension SplashViewController {
+    @IBAction func didTapStartCooking(_ sender: Any) {
+        if let login = Bundle.main.loadNibNamed("LoginViewController", owner: self, options: nil)?.first as? LoginViewController {
+            self.navigationController?.pushViewController(login, animated: true)
+        }
```
Comment (ahmdmhasn) [inline Healthy/Classes/Modules/Splash/SplashViewController.swift]:
```suggestion
        // TODO: Replace with a coordinator 
        let viewModel = LoginViewModel()
        let viewController = LoginViewController(viewModel: viewModel)
        navigationController?.pushViewController(viewController, animated: true)
```

File: Healthy/Classes/testViewController.swift
Comment (ahmdmhasn) [inline Healthy/Classes/testViewController.swift:1]: To be removed

File: Healthy/Classes/Generated/UIImage.Generated.swift (hunk 2 above)
Comment (djayyab) [reply Healthy/Classes/Generated/UIImage.Generated.swift:18]: I did not see any change!
I use icon-...
background-...

File: Healthy/Classes/Modules/Splash/SplashViewController.swift (hunk 3 above)
Comment (djayyab) [reply Healthy/Classes/Modules/Splash/SplashViewController.swift]: I use this way in navigationController but still can't transform to other one

Comment (djayyab) [reply Healthy/Classes/Modules/Splash/SplashViewController.swift]: I used this code before it is not working as we need

File: Healthy/Classes/Generated/UIImage.Generated.swift (hunk 2 above)
Comment (djayyab) [reply Healthy/Classes/Generated/UIImage.Generated.swift:18]: @ahmdmhasn, thank you for your revision😍🤝

**Review by esraakhaled — APPROVED** [review]

**Review by abdallah-7698 — APPROVED** [review]
